src/signals_notebook/entities/tables/table.py

```diff
@@ -268,6 +282,118 @@ def get(self, value: Union[str, UUID], default: Any = None) -> Union[Row, Any]:
             log.debug('KeyError were caught. Default value returned')
             return default
 
+    @classmethod
+    def create(
+        cls,
+        *,
+        container: Container,
+        name: str,
+        content: bytes = b'',
+        content_type: str = ContentType.CSV.value,
```

Comment:
if we dont use .value property we will get this error while sending PATCH request
![image](https://user-images.githubusercontent.com/107862547/181581822-4659dfc7-8c7d-44e7-af66-01d91273419b.png)

Reply:
this bug appears in all Contentful entities while getting ContentType class for entity creation

Reply:
In python 3.11 will appear StrEnum class from enum package.
This class will solve this problem

Reply:
OK